docs/modules/ascii/index.md

```diff
@@ -0,0 +1,58 @@
+---
+title: ASCII-hul Module
+layout: page
+---
+
+# ASCII-hul Module
```

Comment:
Perhaps we can generate some module details and add them here, e.g. current version, date and perhaps even the specifications covered.